### PR TITLE
Add logic for carousel scroll btns

### PR DESCRIPTION
**Pull request by foxwellm**
Angle buttons now shift the carousel over one card length in each direction

### CODE REVIEW

File: src/Components/Carousel.js [hunk 1]
```diff
@@ -3,14 +3,20 @@ import GameCard from './GameCard';
 import { uid } from 'react-uid';
 
 class Carousel extends Component {
+
+  shiftCarousel(event) {
+    const shiftDirection = event.target.classList.contains('fa-angle-right') ? 1 : -1;
+    const currentPosition = event.target.parentElement.childNodes[2].scrollLeft;
```
Comment (foxwellm) [inline src/Components/Carousel.js:9]: Grabbing the parent carousel from whichever button is clicked. scrollLeft gives the position of the element.

File: src/Components/Carousel.js [hunk 2]
```diff
@@ -3,14 +3,20 @@ import GameCard from './GameCard';
...
+    event.target.parentElement.childNodes[2].scrollLeft = currentPosition + (250 * shiftDirection);
+  }
```
Comment (foxwellm) [inline src/Components/Carousel.js:11]: Reassigning the position of the element 250px left or right depending on which button was clicked

**Comment by JustinD85** [comment]
issue #7

File: src/Components/Carousel.js (hunk 2 above)
Comment (whitneyfirth) [reply src/Components/Carousel.js:11]: This is great, nice work Matt. I hadn't used the scrollLeft property yet so it's cool to see it in action.

**Review by whitneyfirth — APPROVED** [review]

File: src/Components/Carousel.js (hunk 1 above)
Comment (JustinD85) [reply src/Components/Carousel.js:9]: Nice use of a ternary here!

**Review by JustinD85 — APPROVED** [review]

**Pull request merged**